frontend/src/styles/tailwind.css

```
@@ -108,92 +108,1548 @@
     white-space: nowrap;
-    border: 0;
   }
 
-  .dashboard-container,
-  .quest-list,
-  .habit-list,
-  #profile-page {
+  .ui-archive-status-dot {
+    width: 0.42rem;
+    height: 0.42rem;
+    background: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)